jira-automation/jira-to-github-processing.mjs

```diff
@@ -80,20 +80,21 @@ export function extractPrioritisationTotalScore(content) {
 
 
 export function jiraToGitHub({issueId, content}) {
+
+    const summary = extractSummary(content)
+    const intendedOutcome = extractIntendedOutcome(content)
+    const impactOnTeams = extractImpactOnTeams(content)
+
     /**
-     * Crime does not use template format so return actual content if crime ticket
+     * Some Crime content does not use template format so return actual content if crime ticket
```

Comment:
Is this something we want to cater for? 
- Would it not be worth getting Crime to follow the template setup & add template if needed

Reply:
Already setup so new issues will have the template format just like in DTS, i.e. it will use the template. There are however lots of older issues we still need to accomodate.